main.py

```diff
@@ -83,26 +77,14 @@ def __init__(self, *args, **kwargs):
         :return:
         """
         log.debug(f"{channel}: <{user}> {message}")
-
+        await super().on_message(channel, user, message)
         if user == config['irc']['nickname']:
```

Comment:
Since the new commands module filters it out based on the **actual** nickname rather than checking the config, this can probably be removed.